src/Microsoft.Framework.DependencyInjection.Abstractions/FrameworkServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Framework.DependencyInjection.Abstractions;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.DependencyInjection.FrameworkExtensions
```

Comment:
I strongly dislike this namespace name, though I can't immediately come up with a better name. Either way, please move the file into a folder that matches the namespace name.